facebook_bot.js

```diff
@@ -99,10 +100,18 @@ if(ops.lt === false && ops.ltsubdomain !== null) {
     process.exit();
 }
 
+var access_token = process.env.page_token;
+try {
+    var access_token = JSON.parse(access_token);
```

Comment:
It's not common to use JSON object as a Environment variable. Can you think of another way to implement this?